airnode-rrp-log/src/main.rs

```diff
@@ -14,13 +15,15 @@ pub struct State {
     // a map of unknown topics
     pub unknown: BTreeMap<H256, H256>,
     pub filtration: LogFiltration,
+    pub pretty_print: bool,
 }
 
 impl State {
-    pub fn new() -> Self {
+    pub fn new(args: &Args) -> Self {
         Self {
             unknown: BTreeMap::new(),
             filtration: LogFiltration::default(),
+            pretty_print: args.pretty_print,
```

Comment:
It looks like `LogFiltration` isn't getting any of the arguments? I've tested by specifying `--by-request-id 0xd2de59b0b33b15b605451480a56977be14137c02aedbd39b258a17c86e0dde3d` and the output is the same as not specifying that filter.

Reply:
its seems you are right, thanks for noticing
It will be fixed

Reply:
Ok great!